file2.js

```diff
@@ -0,0 +1,558 @@
+        teamBoxData = [];
+        jsResponse.data.forEach(function(element) {
+          var option = "<option value='"+ element.id +"'>" + element.text + "</option>";
+          teamBoxData.push(option);
+        });
+        //teams is a array of {id, text} with the id and name of the team
+        $teamBox.html(teamBoxData);
+        if(jsResponse.data.length > 1) {
+          $teamBox.prop("multiple", true);
+          $teamBox.selectpicker("destroy");
+          $teamBox.selectpicker({
+            iconBase: 'fa',
+            tickIcon: 'fa-check'
+          });
+        } else {
+          $teamBox.prop("multiple", false);
+          $teamBox.selectpicker("destroy");
+          $teamBox.selectpicker();
+        }
+
+        $teamBox.prop("disabled", false);
+      };
+
+      var failureFallback = function() {
+        teamBoxData = [];
+      };
+
+      //Gets the teams for the selected organization
+      $orgBox.selectpicker("refresh");
+    };
+
+    ProjectSettings.prototype.bindUpdateProjectName = function(projectId) {
+      $(".BtnUpdateProjectName").click(function() {
+        var url = $(this).data("url");
+        var postVariable = {};
+        postVariable.projectName = $(".TxtUpdateProjectName").val();
+        postVariable.projectId = projectId.toString();
+
+        utils.JSend(url, postVariable, function(data) {
+          $(".ProjectNameFeedback").showSuccess("Saved");
+          location.reload();
+        }, function(error) {
+          $(".ProjectNameFeedback").showError(error);
+        });
+      });
+
+      $(".TxtUpdateProjectName").keyup(function() {
+        $(".BtnUpdateProjectName").removeClass("disabled");
+      });
+    };
+
+    ProjectSettings.prototype.bindUpdateProjectUrl = function(projectId) {
+      $(".BtnUpdateProjectUrl").click(function() {
+        var projectUrlFeedback = $(".ProjectUrlFeedback");
+        var url = $(this).data("url");
+
+        var projectUrl = $(".TxtUpdateProjectUrl").val();
+        var isPrivate = $(this).data("is-private");
+        if (!UrlUtils.getInstance().isValidUrl(projectUrl, isPrivate)) {
+          projectUrlFeedback.showError("The url is invalid");
+          return false;
+        }
+
+        var postVariable = {};
+        postVariable.projectUrl = projectUrl;
+        postVariable.projectId = projectId.toString();
+
+        utils.post(url, postVariable, function(e) {
+          if (e.error) {
+            projectUrlFeedback.showError(e.error);
+          } else {
+            projectUrlFeedback.showSuccess("Saved");
+            location.reload();
+          }
+        }, function() {
+          projectUrlFeedback.showError("Could not change the url, please try again.");
+        });
+      });
+
+      $(".TxtUpdateProjectUrl").keyup(function() {
+        $(".BtnUpdateProjectUrl").removeClass("disabled");
+      });
+    };
+
+    ProjectSettings.prototype.createFileTree = function(jsonData) {
+      var glyph_opts = {
```

Comment:
![Codacy](https://app.dev.codacy.org/assets/images/favicon.png) Issue found: [Identifier 'glyph_opts' is not in camel case. (camelcase)](https://app.dev.codacy.org/app/codacytestuser/CheckPrCommentsPublicProject/pullRequest?prid=8021)